src/ResourceManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement/Generated/Models/UpgradeMode.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
```

Comment:
hi ,
Is this the right version ?

Microsoft (R) AutoRest 0.17.0.0

(after choco install autorest)